ckanext/versioning/fanstatic/versioning.js

```diff
@@ -216,7 +217,12 @@ ckan.module('dataset_versioning_controls', function ($) {
                     if (response.status !== 200) {
                         that._show_error_message(response, 'reverting')
                     } else {
-                        location.href = this._packageUrl;
+                        that.sandbox.notify(
+                            'Success: ',
+                            'Dataset reverted successfully. You can now go back to the main dataset page to see the changes.',
+                            'success'
+                            );
+                        body.scrollTop(0);
```

Comment:
I get the limitations, but it would be nicer UX IMHO if we did something like tell the user that they will be redirected in 5 seconds and do this:

```suggestion
                        body.scrollTop(0);
                        window.setTimeout(function() { location.href = this._packageUrl; }, 5000);
```
(didn't test but something like this).

Just saves most users a click unless they are really impatient. Ugly, but less.

Reply:
I think there can be different ways to implement this, however there is going to be a whole revision of the UX/UI in the upcoming weeks with feedback and suggestion so I tried to just keep it simple for the time being since this change probably won't last too much.